workshop/content/lab2.1_developer.md

```diff
@@ -0,0 +1,63 @@
+# The Developer Stakeholder
+There are many types of users from different types of roles that can all benefit from Serverless. In this section, we will be exploring how Serverless can immediately improve a developer's workflow and productivity.
+
+The first major win is that it frees the developer to simply focus on their application code.  No need to worry about complicated frameworks or application servers that dictate to the developer how they should solve the business problem.
+
+The second major win is that deploying to a development kubernetes environment becomes as easy as a simple one line command. No more messy YAML and a more intuitive workflow that fits the way a developer prefers to work.
+
+1.  Create a dockerhub.com account
+
+2.  Check out the code we will be using.  It is a very simple `Python Flask Hello World` application.
+```
+git clone https://github.com/jkeam/hello-python.git
+cd hello-python
+```
+
+3.  Build the project
```

Comment:
We should use source-to-image instead of docker.  Here's an example: https://github.com/RedHatGov/serverless-workshop-code/tree/main/model

```
GITHUB_URL=https://github.com/RedHatGov/serverless-workshop-code 
oc new-build python:3.6~$GITHUB_URL --name prediction --context-dir=model/prediction
```

If you really want to use docker, we should use `podman` instead.

Reply:
Yeah I don't like creating a dockhub account either.

Reply:
creating github or quay accounts would be fine... if necessary. Avoiding account creation whenever possible is best though.

Reply:
yep good idea.